lms/static/scripts/frontend_apps/index.d.ts

```diff
@@ -0,0 +1,9 @@
+export {};
```

Comment:
Is this needed for a `.d.ts` file?

Also, was there a particular reason for the `index.d.ts` file name? I'm not certain but I think this might override the type of `index.js`, which we don't want. I'd prefer to give it a name like say `types.d.ts` or `onedrive.d.ts` to make it obvious that the types here are not related to the `index.js` file.

Reply:
Yes, I agree.